# HG changeset patch
# User Mike Miller <mtmiller@ieee.org>
# Date 1363465143 14400
#      Sat Mar 16 16:19:03 2013 -0400
# Node ID ba933118869b35cd5f7f58ebe4892924cc960213
# Parent  c69c51983626ae9251cc1592840a462645f8bdb7
build: include GUI translation source files in the distribution tarball

* libgui/Makefile.am (translations): New variable.
(locales): Derive generated names from $(translations).
(EXTRA_DIST): Include $(translations) instead of $(locales).

diff --git a/libgui/Makefile.am b/libgui/Makefile.am
--- a/libgui/Makefile.am
+++ b/libgui/Makefile.am
@@ -21,27 +21,29 @@
 include $(top_srcdir)/build-aux/common.mk
 
 AUTOMAKE_OPTIONS = subdir-objects
 
 MOC_CPPFLAGS =
 
 octlib_LTLIBRARIES = liboctgui.la
 
-locales = \
-  languages/be_BY.qm \
-  languages/de_DE.qm \
-  languages/en_US.qm \
-  languages/es_ES.qm \
-  languages/nl_NL.qm \
-  languages/pt_BR.qm \
-  languages/ru_RU.qm \
-  languages/uk_UA.qm
+translations = \
+  languages/be_BY.ts \
+  languages/de_DE.ts \
+  languages/en_US.ts \
+  languages/es_ES.ts \
+  languages/nl_NL.ts \
+  languages/pt_BR.ts \
+  languages/ru_RU.ts \
+  languages/uk_UA.ts
 
-EXTRA_DIST = default-qt-settings $(locales)
+locales = $(patsubst languages/%.ts, languages/%.qm, $(translations))
+
+EXTRA_DIST = default-qt-settings $(translations)
 CLEANFILES =
 
 BUILT_SOURCES =
 
 noinst_HEADERS =
 
 noinst_LTLIBRARIES =
 
